docs/reference/gc-toolkit/gc-scripts-hub/dataset-uploader.md

```diff
@@ -0,0 +1,115 @@
+
+![Open the GC Dataset Importer app](./images/dataset-uploader-1.jpg)
+
+### 2. Name your dataset
+
+Give your dataset a clear, human-readable name, then click **Check Availability**.
+
+![Enter a dataset name](./images/dataset-uploader-2.jpg)
+
+If available, you’ll see the resulting table name you’ll get in the warehouse.
+
+![Name is available and mapped to table name](./images/dataset-uploader-3.jpg)
+
+### 3. Upload your file
+
+Select your file and click **Upload File**. When the upload completes, proceed to the next step.
+
+![Upload your file and confirm success](./images/dataset-uploader-4.jpg)
+
+**Accepted formats**: CSV, GeoJSON, GPX, JSON, KML, XLS, XLSX, KMZ
+(Shapefiles are not yet supported.)
+
+### 4. (Optional) Choose a known data source
+
+If the file originates from a known tool, toggle **Is this dataset from a known source or tool?** and select it. This adds a `data_source` tag and, for some sources, applies smart formatting.
+
+![Choose a known data source](./images/dataset-uploader-5.jpg)
+
+**Examples**
+
+* **KoboToolbox**: standardizes survey exports for consistent column names and types
+* **CoMapeo / Mapeo / Locus Map**: normalizes fields for observations and waypoints
+
+### 5. Review and finalize
+
+Confirm the summary and click **Save & Finish** to write the data.
+
+![Review summary and save](./images/dataset-uploader-6.jpg)
+
+You’ll see a success message when the import is complete.
+
+![Success confirmation](./images/dataset-uploader-7.jpg)
+
+---
+
+## After You Import
+
+### Use your new table in GC Explorer
+
+Open **GC Explorer** and add your newly created table to a View to explore or publish.
+
+![Add the new table in GC Explorer](./images/dataset-uploader-9.jpg)
+
+See: [GC Explorer](/reference/gc-toolkit/gc-explorer/)
+
+:::info
+
+If you are using [Superset](/reference/gc-toolkit/superset/), or any other tool that uses PostgreSQL as a data source, you can use also access the uploaded data there.
+
+:::
+
+
+### Access your uploaded files in Filebrowser
+
+The importer saves both the **original** file and any **transformed** output in a folder named after your table (e.g., `my_kobo_toolbox_data`).
+
+![Find original and transformed files in Filebrowser](./images/dataset-uploader-8.jpg)
+
+See: [Filebrowser](/reference/gc-toolkit/filebrowser/) for share links or downloads.
+
+:::tip
+
+If you uploaded a spatial data file, you can download it here to use in a tool like [QGIS](/reference/recommended-applications/qgis/).
+
+:::
+
+---
+
+## Tips & Troubleshooting
+
+* **“⚠️ Dataset name is already in usage.”** Pick a different dataset name. The table name must be unique in the warehouse.
+* **Unsupported file type.** Convert it locally to one of the supported formats, then re-upload.
+* **Large files.** If the upload stalls, the file might be too large. Try resizing the file to a smaller size. A recommended maximum size for a file is 5MB on a computer, or 1-2MB on a mobile device.
```

Comment:
Is this a hard limitation?
We will find a lot of times where dataset will be larger.. wouldn't we?
Can't we already find some on our partner's files were that's the case?